bears/c_languages/codeclone_detection/ClangFunctionDifferenceBear.py

```diff
@@ -83,7 +84,7 @@ def get_difference(function_pair,
 class ClangFunctionDifferenceBear(GlobalBear):
     check_prerequisites = classmethod(clang_available)
     LANGUAGES = ClangBear.LANGUAGES
-    REQUIREMENTS = ClangBear.REQUIREMENTS
+    REQUIREMENTS = ClangBear.REQUIREMENTS | {PipRequirement('munkres', '1.0')}
```

Comment:
I believe this should be `munkres3` , per requirements;

```
$ git grep munkres
...
requirements.txt:munkres3==1.*
```

https://pypi.python.org/pypi/munkres (1.0.8) vs https://pypi.python.org/pypi/munkres3 (1.0.5.5)